gcc-4.8/post-link.sh

```diff
@@ -111,6 +111,13 @@ else
         # ... yada yada ... -isystem ${INCDIR}
         sed -i ':a;N;$!ba;s|\(*cpp:\n[^\n]*\)|\1 -isystem '${INCDIR}'|g' "${SPECS_FILE}"
     done
+
+    #
+    # Linux Portability Issue #2.5: linker also needs to find the rest of libc (i.e. libc.so in addition to crtXXX.o)
+    #
+    for library_path in $(/usr/bin/ld --verbose | grep SEARCH_DIR | sed -r 's/SEARCH_DIR\("=?([^"]*)"\);/ \1/g'); do
```

Comment:
Using the full path to the system `/usr/bin/ld` instead of `ld` is a good idea. (It avoids problems if the user has a custom install of binutils.)  While you're at it, can you also fix the call to `ld` on line 49, above?